controllers/articles.controllers.js

```diff
@@ -10,3 +13,13 @@ exports.getArticleById = (request, response, next) => {
       next(error);
     });
 };
+
+exports.getAllArticles = (request, response, next) => {
+  selectAllArticles()
+    .then((articles) => {
+      response.status(200).send({ articles });
+    })
+    .catch((error) => {
```

Comment:
See the error handling notes. There WILL be an error if the code has got to the catch block. An error callback is redundant.